packages/roosterjs-editor-dom/lib/selection/Position.ts

```diff
@@ -146,9 +146,9 @@ export default class Position {
         if (!rect) {
             let { node, element, offset } = this.normalize();
 
-            // 2) if current cursor is inside text node, use range.getClientRects() for safari
+            // 2) if current cursor is inside text node, use range.getClientRects() for safari/chrome
             // or insert a SPAN and get the rect of SPAN for others
-            if (Browser.isSafari) {
+            if (Browser.isSafari || Browser.isChrome) {
```

Comment:
I'm not sure why this is the way this was implemented. I'm not super comfortable with this implementation. Is there any reason we don't test for getClientRects to exists and check that we end up with a unique correct normalized rect when called and fallback on the span method?

I would then remove what I implemented and just do this check so not to rely on user agents.